### DISCUSSION TITLE
Got an unexpected keyword argument when restore `stt_zh_citrinet_512.nemo`.

### DISCUSSION THREAD

**Discussion by Minxiangliu** [NVIDIA-NeMo/NeMo#2849] [Q&A] [answered]
I tried to use `stt_zh_citrinet_512.nemo` for fine-tuning, but an error occurred when updating the vocabulary. 
```
Traceback (most recent call last):
  File "---.py", line 67, in <module>
    model.change_vocabulary(new_tokenizer_dir=TOKENIZER_DIR, new_tokenizer_type="bpe")
TypeError: change_vocabulary() got an unexpected keyword argument 'new_tokenizer_dir'
```
By the way, this error does not appear in the `stt_en_citrinet_512.nemo`.

Any comments here?

**Comment by titu1994** [comment, marked as answer]
This model is not a BPE model - it does not use a tokenizer. Please refer to the documentation of EncDecCTCModel.change_vocabulary()
